### ISSUE TITLE
Check white HCM

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#1775]
Some components do not display properly when the High Contrast Mode is enabled with a light background.

To enable High Contrast Mode with a light background with Windows: 
1. go to Settings > Ease of Access > High Contrast
2. set "Turn on high contrast" to true
3. in "Choose a theme" select the "High Contrast White" option

We need to test all components and create tickets when a display issue is detected.

**Comment by alizedebray** [comment]
https://teams.microsoft.com/l/message/19:meeting_YjllMTQ2NDctMGJhNy00MGMyLTliNzYtN2U0MDg1ZDM4Nzc2@thread.v2/1707816227555?context=%7B%22contextType%22%3A%22chat%22%7D

**Comment by imagoiq** [comment]
- [HCM Light/Dark: Badge checked state is not visible](https://github.com/swisspost/design-system/issues/2641)
- [HCM Light/Dark: Forms invalid and valid feedback + border colors should be kept](https://github.com/swisspost/design-system/issues/2642)
- [HCM Light: Select arrow icon is not visible](https://github.com/swisspost/design-system/issues/2643)
- [HCM Light: Forms elements with disabled attribute are not different from default](https://github.com/swisspost/design-system/issues/2644)
- [HCM Dark/Light: Switch is not usable](https://github.com/swisspost/design-system/issues/2645)
- [HCM Light: Internet Header icons are not displayed](https://github.com/swisspost/design-system/issues/2646)
- [HCM + without: Button Group miss an active state](https://github.com/swisspost/design-system/issues/2647)
- [HCM Dark/Light: Stepper checkmark is not visible](https://github.com/swisspost/design-system/issues/2648)
- [HCM Light: Toast close icon is not visible on hover](https://github.com/swisspost/design-system/issues/2649)
- [HCM Dark/Light: Tooltip arrow has a bigger border ](https://github.com/swisspost/design-system/issues/2650)

Also note that storybook controls as select are not really visible, neither switch boolean control, but there are several issues on the storybook repo linked to that, so we can let Storybook team address those.
